src/Auth/wwwroot/css/site.less

```diff
@@ -0,0 +1,116 @@
+﻿body {
```

Comment:
Are we stuck with LESS here, or could we use SCSS instead?

Reply:
I'll look into it and how it even compiles. I didn't check at all

Reply:
This file is just the source. I think they just compiled the LESS manually and then added a .css.min and a .css min to the project so yes you can switch to SCSS but there is just no compiler whatsoever.

Reply:
Gotcha. In that case, let's keep the CSS but update it to our standards. The .less and the .min.css can go.

Reply:
@misha130 We should definitely be using postCSS here. Take the config from the landing page.